keyboards/paulpcb/readme.md

```diff
@@ -0,0 +1,27 @@
+# paulpcb
+
+![paulpcb](https://i.imgur.com/BZneAHS.jpeg)
+![AWL Paul keyboard](https://i.imgur.com/dRibVfa.jpeg)
+
+The AWL Paul is a reverse Alice style keyboard with an encoder and two 'macro' keys on the RHS. It is designed around simplistic case design that can be 3D printed.
+
+* Keyboard Maintainer: [Alfie_Lockrey](https://github.com/AlfieLockrey)
+* Hardware Supported: PaulPCB V1.0 
+* Hardware Availability: Message me on Discord @ Alfie#6783.**
```

Comment:
```suggestion
* Hardware Availability: Message me on Discord @ Alfie#6783.
```

Reply:
Altered this to a single line, including the line below in a separate commit